### PR TITLE
bump version to 1.21.2

### PR DESCRIPTION
**Pull request by mrocklin**
See also #1808

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+-------------------
+
+-  Fix bug where we didn't check idle/saturated when stealing (:pr:`1801`) `Matthew Rocklin`_
+-  Fix bug where client was noisy when scheduler closed unexpectedly (:pr:`1806`) `Matthew Rocklin`_
+-  Use string-based timedeltas (like ``'500 ms'``) everywhere `Matthew Rocklin`_
+-  Keep logs in scheduler and worker even if silenced (:pr:`1803`) `Matthew Rocklin`_
+-  Support minimum, maximum, wait_count keywords in Adaptive (:pr:`1797`) `Jacob Tomlinson`_ and `Matthew Rocklin`_
+-  Support async protocols for LocalCluster, replace start= with asynchronous= (:pr:`1798`) `Matthew Rocklin`_
+-  Avoid restarting workers when nanny waits on scheduler (:pr:`1793`) `Matthew Rocklin`_
+-  Use ``IOStream.read_into()`` when available `Antoine Pitrou`_
```
Comment (jakirkham) [inline docs/source/changelog.rst]: Is there a PR with this one?

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
 =========
 
+1.21.2 - 2018-03-05
+-------------------
+
+-  Fix bug where we didn't check idle/saturated when stealing (:pr:`1801`) `Matthew Rocklin`_
+-  Fix bug where client was noisy when scheduler closed unexpectedly (:pr:`1806`) `Matthew Rocklin`_
+-  Use string-based timedeltas (like ``'500 ms'``) everywhere `Matthew Rocklin`_
+-  Keep logs in scheduler and worker even if silenced (:pr:`1803`) `Matthew Rocklin`_
+-  Support minimum, maximum, wait_count keywords in Adaptive (:pr:`1797`) `Jacob Tomlinson`_ and `Matthew Rocklin`_
```
Comment (jakirkham) [inline docs/source/changelog.rst]: Might be worth adding PR ( https://github.com/dask/distributed/pull/1800 ) here.

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+1.21.2 - 2018-03-05
+-------------------
+
+-  Fix bug where we didn't check idle/saturated when stealing (:pr:`1801`) `Matthew Rocklin`_
+-  Fix bug where client was noisy when scheduler closed unexpectedly (:pr:`1806`) `Matthew Rocklin`_
+-  Use string-based timedeltas (like ``'500 ms'``) everywhere `Matthew Rocklin`_
```
Comment (jakirkham) [inline docs/source/changelog.rst]: Does this have a PR?

**Comment by jakirkham** [comment]
Looks pretty good. Had a couple questions about changes without a PR ref and one suggestion about a change worth adding.

**Comment by mrocklin** [comment]
Added the other PRs (they were cases where I decided to rebase rather than
squash).  I'm going to opt to avoid 1800 for now.  It's not yet relevant
without future work.

**Comment by jakirkham** [comment]
LGTM. Thanks.

Only mentioned 1800 as it looked interesting. If you would rather not include it, that's ok. :)

**Comment by mrocklin** [comment]
Done manually

**Pull request closed without merging**